libcloudforensics/gcp.py

```diff
@@ -1048,6 +1147,65 @@ def Delete(self):
     self.project.BlockOperation(response)
 
 
+class GoogleComputeImage(GoogleComputeBaseResource):
+  """Class representing a Compute Engine Image.
```

Comment:
Add new line before `Attributes:`

Reply:
no new attributes for this class. just the inherited ones.